lib/dependencies.sh

```diff
@@ -88,7 +88,7 @@ yarn_node_modules() {
 
   echo "Installing node modules (yarn.lock)"
   cd "$build_dir" || return
-  monitor "yarn-install" yarn install --production="$production" --frozen-lockfile --ignore-engines 2>&1
```

Comment:
`--production`, `--frozen-lockfile` and `ignore-engines` cli options are not valid anymore